content/en/docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -47,7 +47,7 @@ when using GPUs:
   * You can specify GPU in both `limits` and `requests` but these two values
     must be equal.
   * You cannot specify GPU `requests` without specifying `limits`.
-- Containers (and pods) do not share GPUs. There's no overcommitting of GPUs.
+- Containers (and Pods) do not share GPUs. There's no overcommitting of GPUs.
 - Each container can request one or more GPUs. It is not possible to request a
```

Comment:
Do you need the parens?

Reply:
I'll leave these from the previous content